monai/deploy/operators/stl_conversion_operator.py

```diff
@@ -0,0 +1,443 @@
+@md.env(pip_packages=["numpy>=1.21", "numpy-stl>=2.12.0", "scikit-image>=0.17.2", "trimesh>=3.8.11"])
+class STLConversionOperator(Operator):
+    """Converts volumetric image to surface mesh in STL format, file output only."""
+
+    def __init__(
+        self, output_file=None, class_id=None, is_smooth=True, keep_largest_connected_component=True, *args, **kwargs
+    ):
+        """ "Creates an object to generate surface mesh in a STL file if the path is provided.
+
+        Args:
+            output_file (str, optional): output stl file relative path. Default to None for no file output.
+            class_id (array, optional): Class label ids. Defaults to None.
+            is_smooth (bool, optional): smoothing or not. Defaults to True.
+            keep_largest_connected_component (bool, optional): Defaults to True.
+        """
+        super().__init__(*args, **kwargs)
+        self._logger = logging.getLogger("{}.{}".format(__name__, type(self).__name__))
+        self._class_id = class_id
+        self._is_smooth = is_smooth
+        self._keep_largest_connected_component = keep_largest_connected_component
+        self._output_file = output_file if output_file and len(output_file) > 0 else None
+
+        self._converter = STLConverter(*args, **kwargs)
+
+    def compute(self, op_input: InputContext, op_output: OutputContext, context: ExecutionContext):
+        """Gets the input (image), processes it and sets results in the output.
+
+        When used in a leaf operator, this function cannot set its output as in-memory object due to
+        current limitation, and only file output, for DataPath IOType_DISK, will be saved in the
+        op_output path, which is mapped to the application's output path by the execution engine.
+
+        Args:
+            op_input (InputContext): An input context for the operator.
+            op_output (OutputContext): An output context for the operator.
+            context (ExecutionContext): An execution context for the operator.
+        """
+
+        input_image = op_input.get("image")
+        if not input_image:
+            raise ValueError("Input is None.")
+
+        op_output_config = op_output.get()
+        if self._output_file and len(self._output_file) > 0:
+            # The file output folder is either the op_output or app's output depending on output types.
+            output_folder = (
+                op_output_config.path if isinstance(op_output_config, DataPath) else context.output.get().path
+            )
+            self._output_file = output_folder / self._output_file
+            self._output_file.parent.mkdir(exist_ok=True)
+            self._logger.info(f"Output will be saved in file {self._output_file}.")
+
+        stl_bytes = self._convert(input_image, self._output_file)
+
+        if not isinstance(op_output_config, DataPath):
+            op_output.set(stl_bytes)
+
+    def _convert(self, image: Image, output_file: Union[Path, None] = None):
```

Comment:
it would be same with Optional.
```suggestion
    def _convert(self, image: Image, output_file: Optional[Path] = None):
```